src/pages/builders/faq.tsx

```diff
@@ -71,12 +71,45 @@ const buildersFaqContent = (urls: BuildersFaqUrls): FaqContent[] => [
       </div>
     ),
     answer: (
-      <p>
-        When the submission period comes, we&apos;ll open a form to send a detailed presentation &
-        links to your contribution. It can be docs, code, proof of concepts... If it seems cool,
-        you&apos;ll be nominated by the judges and be invited to discuss your contribution with the
-        team.
-      </p>
+      <>
+        <p>
+          Follow the &quot;How to submit&quot; instructions provided for each challenge. Sometimes a
+          transaction you submit on-chain is enough; other times, you should tweet a thread with
+          specific hashtags. We may open a form to send a detailed presentation & links to your
+          contribution. It can be docs, code, proof of concepts...
+        </p>
+        <p>
+          For most of the challenges, you&apos;ll be nominated by the judges. You might be invited
+          to discuss your contribution with the team.
+        </p>
+      </>
+    )
+  },
+  {
+    question: (
+      <div className="okp4-nemeton-web-tasks-accordion-title">
+        <h3>How do I know I completed a challenge? When will I receive rewards?</h3>
+      </div>
+    ),
+    answer: (
+      <>
+        <p>
+          For each challenge, the OKP4 team will communicate eligible wallets a few days after the
+          end period. POAP badges (Stargaze NFT) will be sent at this moment, but $KNOW token will
```

Comment:
You could put a link to the Stargaze (https://www.stargaze.zone/).

Reply:
I will add the link in another PR.